src/metrics/listener.spec.ts

```diff
@@ -96,6 +100,40 @@ describe("MetricsListener", () => {
 
     expect(spy).toHaveBeenCalledWith(`{"e":1487076708,"m":"my-metric","t":["tag:a","tag:b"],"v":10}\n`);
   });
+  it("sends metrics to statsD when extension is enabled", async () => {
```

Comment:
I would reword this so that it is clear that this should happen EVEN WHEN log forwarding is enabled. I.e. the behavior we need to test for is that log forwarding does not happen when the extension is enabled.